Modules/ConfigOptions.lua

```diff
@@ -718,23 +718,23 @@ return {
 		modList:NewMod("Condition:KilledUniqueEnemy", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
 	end },
 	{ var = "conditionHaveArborix", type = "check", label = "Do you have Iron Reflexes?", ifCond = "HaveArborix", tooltip = "This option is specific to Arborix",apply = function(val, modList, enemyModList)
-		modList:NewMod("Condition:HaveIronReflexes", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
-		modList:NewMod("Keystone", "LIST", "Iron Reflexes", "Config")
+		modList:NewMod("Condition:HaveIronReflexes", "FLAG", true, "Config", { type = "Condition", var = "Combat" }, { type = "Condition", var = "HaveArborix" })
+		modList:NewMod("Keystone", "LIST", "Iron Reflexes", "Config", { type = "Condition", var = "HaveArborix" })
```

Comment:
are these changes related?

Reply:
Nope, these are meant to be part of another pull request